google/cloud/storage/client.h

```diff
@@ -238,6 +238,24 @@ class Client {
     raw_client_->DeleteObject(request);
   }
 
+  /**
+   * Retrieve the list of BucketAccessControls for a bucket.
```

Comment:
The GSG says function comments should be descriptive ("Retrieves the list …") rather than imperative ("Retrieve the list …"). Should the project style guide highlight that you're not following that?

Reply:
Ooops.  My bad.  I will fix this one.  Created #968 to fix this elsewhere.